nevergrad/optimization/optimizerlib.py

```diff
@@ -1511,6 +1511,44 @@ def _internal_tell_candidate(self, candidate: p.Parameter, loss: tp.FloatLoss) -
         self._optim.tell(candidate, loss)
 
 
+@registry.register
+class MetaModelOpO(base.Optimizer):
```

Comment:
Not used anywhere for now: let's see if it finds a role. I think this can be implemented in a shorter manner by specifying multivariate_optimizer (I feel that @jrapin will prefer that :-) ).

Reply:
I agree, as I wrote on workchat :) I will improve it soon.